### PR TITLE
Bug/compacted button

### PR DESCRIPTION
**Pull request by Sven65**
## Explain the changes you’ve made

I've stopped buttons from collapsing, according to #CU-2qa0muq

## Explain your solution

I literally changed a `1` to a `0`.

## How to test

Concrete example:

1. Checkout this branch
2. Start a case
3. Go to the salary step
4. Click the "Lägg till lön" button
5. Exit the modal
6. See that the button doesn't collapse

## Tested environments

- [] Storybook on a iOS device/simulator.
- [x] Building the Application on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.

## Other notes

This took me 3 hours of debugging, only to find out that it was a styling issue.

... Man, I hate css.

---

## Checklist for reviewers

Here are a few points that are useful for reviewers to check.

The short list of the (maybe) most important points are:

- Does this PR work as expected?
- Does the PR properly only concern a single feature/bug/thing?
- Is the code clear and concise?
- Will the changes impact existing functionality?
- Are there any linting errors (ESLint) that should be resolved?
- Do the automatic GitHub checks pass?

However it is good if a reviewer can be as thorough as possible,
within reasonable limits of course.

### Functionality

- Does this PR work as expected?
- If this is a new feature, does it add value, or is it a sign of feature-creep?
- Will the changes impact existing functionality?
- Does the changes create inconsistencies somewhere else?
- Does the code work on different platforms if applicable?
- How does the code handle if unexpected values are passed to functions etc.?

### Readability, syntax, formatting, style

- Is the code clear and concise?
- Has styling rules (Prettier) been applied to included files?
- Are functions/variables/etc. aptly named?

### Design

- Is the code properly planned and designed?
- Is code organized/separated in a logical fashion?
- Does the code follow clean code practices to a reasonable degree?
- Is the code self-documenting enough?
- Is the code in sync with existing patterns/technologies?
- Does the code reuse existing functionality where applicable?
- Is the code designed with reusability in mind?
- Does the code have hard-coded values that can be refactored?

### Patterns, idioms, tests, best practices

- Are there any linting errors (ESLint) that should be resolved?
- Is the code properly typed (TypeScript)?
- Does the code utilize language features to their full potential?

### Debugging, testing, configuring

- Is appropriate logging used?
- Have tests been added/modified to accommodate the changes?
- Is the feature sufficiently configurable in case tweaks are required in the future?

### Performance, reliability, scalability

- Is the code well optimized?
- Does the code scale?
- Does the code handle failure scenarios?

### Security

- Is there any obvious potential for exploits and security threats?
- Does code check proper authorization where applicable?

### Etiquettes

- Are commits atomic (does only one thing per commit)?
- Does the PR properly only concern a single feature/bug/thing?
- Are commit messages well-written?
- Do the automatic GitHub checks pass?

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-2qa0muq Knapp för att lägga till ny post blir ihopfälld](https://app.clickup.com/t/2qa0muq)

File: source/components/atoms/Button/Button.tsx [hunk 1]
```diff
@@ -231,6 +231,8 @@ const Button = (props) => {
     return child;
   });
 
+  console.log("childre", children);
```
Comment (kajm) [inline source/components/atoms/Button/Button.tsx:234]: remove console.log

**Review by kajm — CHANGES_REQUESTED** [review]

**Review by dannil76 — APPROVED** [review]

File: source/components/molecules/NavigationButtonField/NavigationButtonField.tsx [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+import styled from "styled-components/native";
+import { Button, Icon, Text } from "../../atoms";
+import Box from "../../atoms/Box/Box";
 
 const ButtonFieldWrapper = styled(Box).attrs({
-  m: '2px',
+  m: "2px",
 })`
-  flex: 1;
+  flex: 0;
```
Comment (kajm) [inline source/components/molecules/NavigationButtonField/NavigationButtonField.tsx:10]: Do we really need the `flex` property at all?

File: source/components/molecules/NavigationButtonField/NavigationButtonField.tsx [hunk 3]
```diff
@@ -66,7 +66,7 @@ const NavigationButtonField: React.FC<Props> = ({
     <ButtonFieldWrapper>
       <Button variant="outlined" onClick={onClick} colorSchema={colorSchema}>
         {iconName.length ? <Icon name={iconName} /> : null}
-        {text && <Text>{text}</Text>}
+        {text ? <Text>{text}</Text> : null}
```
Comment (kajm) [inline source/components/molecules/NavigationButtonField/NavigationButtonField.tsx:69]: What is the reason for returning `null` ?

**Review by kajm — CHANGES_REQUESTED** [review]

File: source/components/molecules/NavigationButtonField/NavigationButtonField.tsx (hunk 3 above)
Comment (Sven65) [reply source/components/molecules/NavigationButtonField/NavigationButtonField.tsx]: Missed restoring it after debugging

File: source/components/molecules/NavigationButtonField/NavigationButtonField.tsx (hunk 2 above)
Comment (Sven65) [reply source/components/molecules/NavigationButtonField/NavigationButtonField.tsx]: Turns out we don't!

**Review by kajm — APPROVED** [review]

**Pull request merged**